src/main/kotlin/blackjack/application/BlackjackService.kt

```diff
@@ -1,16 +1,17 @@
 package blackjack.application
 
-import blackjack.domain.Deck
+import blackjack.domain.Bet
 import blackjack.domain.Game
 import blackjack.domain.Players
 
 class BlackjackService {
-    fun initializeGame(
-        names: List<String>,
-        deck: Deck,
-    ): Game {
-        val players = Players.from(names)
-        val game = Game(players, deck)
+    fun createGame(command: CreateGameCommand): Game {
+        val bets = command.bets.map { Bet(it) }
```

Comment:
사소하지만 아래와 같이 작성해볼 수도 있겠네요!
```suggestion
        val bets = command.bets.map(::Bet)
```